### PR TITLE
Tweak animation file names to be less confusing in Creating the player scene

### PR DESCRIPTION
**Pull request by Calinou**
- Follow-up to https://github.com/godotengine/godot-docs/pull/9958.

It's more verbose, but it's easier to understand and doesn't rely on the reader knowing how to interpret the `[1/2]` part.

- See https://github.com/godotengine/godot-docs-user-notes/discussions/105#discussioncomment-10967761.

### CODE REVIEW

File: getting_started/first_2d_game/02.player_scene.rst [hunk 1]
```diff
@@ -69,12 +69,13 @@ Click on the ``SpriteFrames`` you just created to open the "SpriteFrames" panel:
...
-"walk". Then click the "Add Animation" button to create a second animation named
-"up". Find the player images in the "FileSystem" tab - they're in the ``art``
+On the left is a list of animations. Click the ``default`` one and rename it to
+``walk``. Then click the **Add Animation** button to create a second animation named
+``up``. Find the player images in the FileSystem dock - they're in the ``art``
 folder you unzipped earlier. Drag the two images for each animation, named
-``playerGrey_walk[1/2]`` and ``playerGrey_walk[2/2]``, into the "Animation Frames"
-side of the panel for the corresponding animation:
+``playerGrey_walk1``, ``playerGrey_walk2`` (for ``walk``),
+``playerGrey_up1`` and ``playerGrey_up2`` (for ``up``) to the **Animation Frames**
```
Comment (AThousandShips) [inline getting_started/first_2d_game/02.player_scene.rst:77]:
```suggestion
``playerGrey_up1``, and ``playerGrey_up2`` (for ``up``) to the **Animation Frames**
```

**Review by Piralein — COMMENTED** [review]
It still reads confusing for me, maybe a bullet list works better in this case

```suggestion
On the left is a list of animations. Click the ``default`` one and rename it to
``walk``. Then click the "Add Animation" button to create a second animation named
``up``.

Find the player images in the FileSystem dock - they're in the ``art``
folder you unzipped earlier.
Drag the two images for each animation,
into the **Animation Frames** side of the panel for the corresponding animation:

* ``playerGrey_walk1`` and ``playerGrey_walk2`` for the ``walk`` animation
* ``playerGrey_up1`` and ``playerGrey_up2`` for the ``up`` animation
```

or maybe the ``and`` between the 2 animations

**Comment by mhilbrunner** [comment]
Thanks for following up on this, when cherry-picking the original I kinda agreed this could be improved further :)

**Comment by tetrapod00** [comment]
We just got a duplicate PR (https://github.com/godotengine/godot-docs/pull/10397) to fix the bug introduced in https://github.com/godotengine/godot-docs/pull/9958. So I'm tempted to merge this as-is as a bugfix.

**Review by tetrapod00 — APPROVED** [review]
It's a bugfix, and it's correct and readable as-is. If we want to make additional changes like Piralein's suggestion, it's possible to do that in the future.

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Seems reasonable, merged. Thanks!

**Pull request merged**